docs/csharp/language-reference/compiler-options/target-appcontainerexe-compiler-option.md

```diff
@@ -13,7 +13,7 @@ If you use the **-target:appcontainerexe** compiler option, the compiler creates
 ```  
   
 ## Remarks  
- To require the app to run in an app container, this option sets a bit in the [Portable Executable](https://msdn.microsoft.com/library/windows/desktop/ms680547(v=vs.85).aspx?id=19509) (PE) file. When that bit is set, an error occurs if the CreateProcess method tries to launch the executable file outside an app container.  
+ To require the app to run in an app container, this option sets a bit in the [Portable Executable](/windows/desktop/Debug/pe-format).aspx?id=19509) (PE) file. When that bit is set, an error occurs if the CreateProcess method tries to launch the executable file outside an app container.  
```

Comment:
`[Portable Executable](/windows/desktop/Debug/pe-format).aspx?id=19509)`

There is one opening `(`, but two closing `)`s The `.aspx?id=19509)` is rendered as text after the link